lib/components/TodoList.tsx

```diff
@@ -1,80 +1,107 @@
-import { nanoid } from "nanoid";
-import React, { useState } from "react";
-import { ReadTransaction, Replicache } from "replicache";
-import { useSubscribe } from "replicache-react";
-import { M } from "../mutators";
-import { Todo, Update } from "../types";
-import { DeleteIcon } from "./DeleteIcon";
+import type { Todo } from '@/dbschema/interfaces'
```

Comment:
Small ask: can we format all of these files with prettier with the default config? I know it was a little loosely formatted before, but seeing all of thse whitespace changes that are going actually in the arguably "wrong" direction is making it a little harder to review the actual changes.

Reply:
Ah! I have no idea how, but Prettier has been picking up on some config I may have in VS Code, instead of using its defaults.

Now formatted as before ;)